prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -528,11 +536,20 @@ class LookupTableCreateCommand : GenericLookupTableCommand(
     private val tableName by option("-n", "--name", help = "The name of the table to perform the operation on")
         .required()
 
+    /**
+     * Force to create table(s).
+     */
+    private val force by option("-f", "--force", help = "Force to create the table").flag()
```

Comment:
Maybe more context what the force does.  Force the creation of a new table even if it is a duplicate?

Reply:
Done.